src/components/Heading/index.js

```diff
@@ -0,0 +1,47 @@
+import React, {PropTypes} from 'react'
+
+export const levels = ['1', '2', '3', '4', '5']
```

Comment:
But yes I also use it for dynamically rendering different elements and styles with the `outputByLevel[level]` call. This is now in `egghead-ui` and will be deleted from this project once I can use the egghead-ui version.